fuzz/fuzz-read-print-write.cpp

```diff
@@ -27,6 +27,11 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t * data, size_t size) {
     // produce lots of garbage on stdout.
     std::ostringstream buffer;
     image->printStructure(buffer, Exiv2::kpsNone);
```

Comment:
line 24, seems to not do anything.  `exifData()` just returns a reference to the container holding the exif metadata.

Do you think it would be helpful to the printing of the actual metadata values here as well? 
That would cover all the functions we use to convert various raw metadata into their respective string representation.  

Unfortunately, this isn't directly implemented in the image class, but in [`printMetadata()`](https://github.com/Exiv2/exiv2/blob/main/src/actions.cpp#L440) in `actions.cpp`. 
But we could implement something similar relatively easily here since we don't need to worry about options and could just print every single metadatum we find.

Reply:
Good catch. It was in the [original version](https://github.com/Exiv2/exiv2/blob/b2cdf2a535f399333f80519e54501be224bdb1ed/fuzz/read-metadata.cpp#L14) fuzzer and I kept it because I didn't realize that it doesn't do anything.